so.libdb.dissent.yml

```diff
@@ -0,0 +1,241 @@
+id: so.libdb.dissent
+runtime: org.gnome.Platform
+runtime-version: '45'
+sdk: org.gnome.Sdk
+command: dissent
+
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.golang
+
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --socket=pulseaudio
+  - --share=ipc
+  - --share=network
+  - --talk-name=org.freedesktop.Notifications
+  - --talk-name=org.freedesktop.secrets
+  - --talk-name=org.kde.kwalletd5
+  - --talk-name=org.kde.StatusNotifierWatcher
+  - --filesystem=xdg-run/keyring  # TODO: Check if this is even needed
+  - --device=dri
+
+modules:
+  - name: dissent
+    buildsystem: simple
+    build-options:
+      append-path: /usr/lib/sdk/golang/bin
+      env:
+        GOBIN: /app/bin
+        GOROOT: /usr/lib/sdk/golang
+    build-commands:
+      - go install -v -mod=vendor ./...
+      - desktop-file-edit --set-icon=$FLATPAK_ID --remove-key=DBusActivatable nix/$FLATPAK_ID.desktop
+      - install -Dm644 nix/$FLATPAK_ID.desktop /app/share/applications/$FLATPAK_ID.desktop
+      - install -Dm644 internal/icons/hicolor/scalable/apps/logo.svg /app/share/icons/hicolor/scalable/apps/$FLATPAK_ID.svg
+      - install -Dm644 $FLATPAK_ID.metainfo.xml /app/share/metainfo/$FLATPAK_ID.metainfo.xml
+    sources:
+      - type: archive
+        url: https://github.com/diamondburned/dissent/archive/refs/tags/v0.0.19.tar.gz
+        sha256: 38fe04773f85e522be5fd42b2aac2d106fd93baf0b5b53b40f66f876b1ba38e3
+
+      - type: file
+        url: https://raw.githubusercontent.com/diamondburned/dissent/b04fc22f2e3bf2937d72b63a9b01b1c1943c2eae/so.libdb.dissent.metainfo.xml
+        sha256: 1a3e23fcbccc436cb0b15d004b64396489dd5daace57257457df0340af386eab
+
+      # Workaround for Go modules generated by github.com/dennwc/flatpak-go-mod
+      - type: file
+        path: modules.txt
+        dest: vendor
+
+      - type: archive
+        url: https://proxy.golang.org/github.com/diamondburned/adaptive/@v/v0.0.2-0.20221227093656-fa139be203a8.zip
+        strip-components: 3
+        dest: vendor/github.com/diamondburned/adaptive
```

Comment:
These should be separated from the main manifest, you are auto generating them.

Reply:
How can I do that?

Reply:
See https://github.com/flathub/io.github.dweymouth.supersonic/blob/master/go.mod.yml or https://github.com/flathub/com.visualstudio.code.tool.git-lfs/blob/branch/23.08/vendor-sources.json